samples/README.md

```diff
@@ -0,0 +1,4 @@
+# samples
+
+It's the samples of ServiceComb Toolkit
+1. toolkit-maven-plugin-sample
```

Comment:
Content and implementation do not match,has the description been written yet?

Reply:
Thanks for your reminder, I forgot to update the README.md and the content has expired. I will update it in the next commit.